api.md

```diff
@@ -187,6 +202,16 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+First, create a multinode with two data nodes, then convert table `conditions` to 
+a distributed hypertable with just time partitioning on column `time`, 
+finally add a space partitioning on `location` with two partitions:
```

Comment:
```suggestion
```suggestion
Now in a multi-node example for distributed hypertables, 
setup your cluster to have one access node and two data nodes, 
and configure your access node for access to the data nodes.
Then, convert your table `conditions` to 
a distributed hypertable with just time partitioning on column `time`, 
and finally add a space partitioning dimension on `location`.  
By default, the `number_partitions` will be set to the number of data nodes
in the distributed hypertable `conditions`.
```